x/concentrated-liquidity/types/msgs.go

```diff
@@ -157,3 +157,36 @@ func (msg MsgCollectIncentives) GetSigners() []sdk.AccAddress {
 	}
 	return []sdk.AccAddress{sender}
 }
+
+var _ sdk.Msg = &MsgCreateIncentive{}
+
+func (msg MsgCreateIncentive) Route() string { return RouterKey }
+func (msg MsgCreateIncentive) Type() string  { return TypeMsgCollectIncentives }
+func (msg MsgCreateIncentive) ValidateBasic() error {
```

Comment:
should we check if min_uptime is valid here too? basically the same logic as https://github.com/osmosis-labs/osmosis/pull/4519/files#diff-8fb13ed7a62b35b89555bdc906798a4353c391186de40eacadac4045c1e4d7bdR526-R534

Reply:
Agreed!